### PR TITLE
[bitnami/phpbb] Add tests and publish using VIB

### PR DESCRIPTION
**Pull request by joancafom**
**Description of the change**
The main objective of this PR is to publish our Bitnami PhpBB Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

**Benefits**

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

**Possible drawbacks**
Automated tests could show to be potentially flaky. 

**Test results**

Tested in https://github.com/joancafom/charts/pull/62

### Checklist

- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in the agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/phpbb/cypress/cypress.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "baseUrl": "http://localhost",
+  "defaultCommandTimeout": 30000,
```
Comment (agomezmoron) [inline .vib/phpbb/cypress/cypress.json:3]: Which is the aim of it?

File: .vib/phpbb/cypress/cypress/integration/phpbb_spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows to access Administrator Control Panel', () => {
+  cy.login();
+  cy.contains('Administration Control Panel').click();
+  // ACP requires double login
+  cy.get('[type="password"]').type(Cypress.env('password'));
+  cy.get('[name=login]').click();
+  cy.contains('ACP Logout');
```
Comment (agomezmoron) [inline .vib/phpbb/cypress/cypress/integration/phpbb_spec.js:10]: IMHO: I would simplify onlwith with "Logout"

**Review by agomezmoron — APPROVED** [review]
LGTM. I added minor comments but it's fine :)

**Review by FraPazGal — APPROVED** [review]
LGTM!

File: .vib/phpbb/cypress/cypress/integration/phpbb_spec.js (hunk 2 above)
Comment (joancafom) [reply .vib/phpbb/cypress/cypress/integration/phpbb_spec.js:10]: Unfortunately, we can't do that. The UI has two different elements: `Logout` and `ACP Logout` (regular logout from the system vs. Administration Control Panel Logout).

This is why the whole text is here 😁

File: .vib/phpbb/cypress/cypress.json (hunk 1 above)
Comment (joancafom) [reply .vib/phpbb/cypress/cypress.json:3]: The aim is to reduce flakiness, we usually add this to other assets too.

**Pull request merged**